contrib/startup_regtest.sh

```diff
@@ -153,39 +153,58 @@ start_ln() {
 
 fund_ln() {
 
+	if [ -z "$1" ]; then
+		node1=1
+	else
+		node1="$1"
+	fi
+
+	if [ -z "$2" ]; then
+		node2=2
+	else
+		node2="$2"
+	fi
+
```

Comment:
I think we want a generic "fund_nodes" helper instead?  Which does all nodes if no args, otherwise the listed nodes?

Reply:
fund_nodes does all nodes or listed ones! Also looks for `--wallet=...`